src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagLevel.cs

```diff
@@ -3,25 +3,25 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+namespace Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag
 {
     /// <summary>
-    /// Level of a custom tag.
+    /// Level of a extended query tag.
     /// </summary>
-    public enum CustomTagLevel
```

Comment:
I'm thinking if we name it "QueryTagLevel" instead for accuracy, since it's also apply to Core tags.